src/icon/internal.tsx

```diff
@@ -17,18 +17,16 @@ type InternalIconProps = IconProps &
   };
 
 function iconSizeMap(height: number) {
-  if (height) {
-    if (height >= 50) {
-      return 'large';
-    } else if (height >= 36) {
-      return 'big';
-    } else if (height >= 24) {
-      return 'medium';
-    } else if (height <= 16) {
-      return 'small';
-    } else {
-      return 'normal';
-    }
+  if (height >= 50) {
```

Comment:
Do we want to add a test for using `renderToStaticMarkup` to confirm the behavior?

Reply:
+1. Please check how other `@jest-environment node` tests work and let's do the same here: https://github.com/cloudscape-design/components/search?q=jest-environment+node